packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.ts

```diff
@@ -58,7 +60,21 @@ export default class CardPayCreateMerchantWorkflowPrepaidCardChoiceComponent ext
     this.merchantRegistrationFee = yield this.layer2Network.strategy.merchantRegistrationFee();
   }
 
+  get prepaidCards() {
+    return this.layer2Network.safes.value.filterBy(
+      'type',
+      'prepaid-card'
+    ) as PrepaidCardSafe[];
+  }
```

Comment:
Should this exclude prepaid cards whose balance is less than the required amount of SPEND? ([This PR](https://github.com/cardstack/cardstack/pull/2027) should help with that)

Reply:
We will need to fade them out / make them not selectable